### DISCUSSION TITLE
Add ability to reorder locations

### DISCUSSION THREAD

**Discussion by RyloRiz** [spacedriveapp/spacedrive#2324] [Feature Request]
<img width="166" alt="Screenshot 2024-04-14 at 10 48 58 AM" src="https://github.com/spacedriveapp/spacedrive/assets/91710711/9f2a0e38-8fcd-472e-ba3d-36a8e6e8969a">

I should be able to reorder Locations in the sidebar (which should reflect on the Overview page as well).

**Comment by utkubakir** [comment]
This is definitely planned, same for tag ordering too.
